### PR TITLE
Update 16-mdjaniga.md

### PR DESCRIPTION
**Pull request by mdjaniga**

### CODE REVIEW

File: _slides/16-mdjaniga.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+No Clue What I Am!
```
Comment (amd-juharris) [inline _slides/16-mdjaniga.md:6]: Clearly this is a gold mining octo cat.  I don't have any better idea for a caption though.

File: _slides/16-mdjaniga.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
```
Comment (amd-juharris) [inline _slides/16-mdjaniga.md:3]:
```suggestion
title: "Welcome to 16-mdjaniga's slide!"
```

**Review by amd-juharris — APPROVED** [review]
Finished the review.

File: _slides/16-mdjaniga.md (hunk 1 above)
Comment (mdjaniga) [reply _slides/16-mdjaniga.md:6]: "Eureka!"???

**Pull request merged**